pkg/configuration/config.go

```diff
@@ -65,6 +65,12 @@ func Setup() {
 		}
 	}
 
+	// This may be different from `UserConfigDir` if `--configuration` was provided
+	configDir := filepath.Dir(UserConfigFile)
+	if !utils.Exists(configDir) {
+		utils.HandleError(fmt.Errorf("Parent directory must exist when providing a custom config file path: %s", configDir))
```

Comment:
I don't want to rely on implicit knowledge that the path must've been overridden, unless we're explicitly checking that a custom path was used
```suggestion
		utils.HandleError(fmt.Errorf("Configuration file directory does not exist %s", configDir))
```

Reply:
Good point 👍 I'll make the change as part of my commit amend